app/javascript/components/Nav.tsx

```diff
@@ -101,37 +108,52 @@ type Props = {
   children: React.ReactNode;
   title: string;
   footer: React.ReactNode;
-  compact?: boolean;
 };
 
-export const Nav = ({ title, children, footer, compact }: Props) => {
+export const Nav = ({ title, children, footer }: Props) => {
   const [open, setOpen] = React.useState(false);
   const close = React.useCallback((): void => setOpen(false), []);
   const toggle = React.useCallback((): void => setOpen((prev) => !prev), []);
-  const contextValue = React.useMemo<NavContextValue>(() => ({ close }), [close]);
+  const contextValue = React.useMemo<NavContextValue>(() => ({ open, close }), [open, close]);
 
   return (
     <NavContext.Provider value={contextValue}>
-      <nav aria-label="Main" className={classNames("main-nav", { compact, open })}>
-        <div className="navbar">
-          <a href={Routes.root_url()}>
-            <span className="logo-g">&nbsp;</span>
+      <nav
+        aria-label="Main"
+        className={classNames(
+          "flex flex-col overflow-x-hidden overflow-y-auto bg-black text-nav-foreground lg:static lg:w-[12.8125rem]",
+          {
+            "fixed z-10 size-full lg:static lg:w-[12.8125rem]": open,
+          },
+        )}
+      >
+        <div className="override grid grid-cols-[auto_1fr_auto] items-center gap-3 p-4 text-[1.125rem] leading-[1.4] lg:hidden">
```

Comment:
Please don't use values like `12.8125rem`, `1.125rem`, etc - we can use the nearest convenient Tailwind equivalent